Packs/CommonScripts/Scripts/URLSSLVerification/URLSSLVerification.py

```diff
@@ -30,48 +31,79 @@ def arg_to_list_with_regex(arg):
     return arg
 
 
+def verify_ssl_certificate(url):
+    """
+    Verifies the SSL certificate of a given URL by following any redirects and checking the certificate for each redirected URL.
+    If all of the redirect chain are all http return malicious message.
+
+    Args:
+        url (str): The URL to verify.
+
+    Returns:
+        dict: A dictionary with  Information and a description of the issue if the SSL certificate verification.
+              Returns None if the verification is successful.
+    """
+    try:
+        response = requests.get(url, timeout=5, allow_redirects=True)
+        redirect_chain = response.history + [response]
+    except requests.exceptions.SSLError:
+        return {"Vendor": VENDOR, "Description": "SSL Certificate verification failed"}
+    except requests.exceptions.RequestException:
+        return {"Vendor": VENDOR, "Description": "Failed to establish a new connection with the URL"}
+
+    is_all_http = True
+    for resp in redirect_chain:
+        redirected_url = resp.url
+        if not redirected_url.startswith(SSL_PREFIX):
+            continue
+        is_all_http = False
+        try:
+            requests.get(redirected_url, timeout=5, verify=True)
```

Comment:
No need for this duplication. 
Write a helper function with the call and use it in both times